docs/releaseNotes.md

```diff
@@ -0,0 +1,87 @@
+the twitter future API to a separate module and replace the core API with Java8 CompletableFuture. It reduces the dependencies introduced
+by Scala and also avoid maintaining multiple versions for different scala versions.
+
+### Better buffer/memory management
+
+After upgrading to BookKeeper 4.5.0, we update the buffer and memory management in distributedlog to leverage netty's buffer management.
+It reduces object allocation, memory copies and improves the latency because less jvm garbages are produced.
+
+
+## Full list of changes
+
+### JIRA
+
+
+####Sub-task
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-208'>DL-208</a>] -         Update release notes about the download location
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-209'>DL-209</a>] -         Update build script to build distributedlog website under `bookkeeper.apache.org/distributedlog`
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-210'>DL-210</a>] -         Update DL mailing lists
+</li>
+</ul>
+                            
+####Bug
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-19'>DL-19</a>] -         Can&#39;t compile thrift file
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-124'>DL-124</a>] -         Use Java8 Future rather than twitter Future
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-128'>DL-128</a>] -         All TODO items should have jira tickets.
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-195'>DL-195</a>] -         Apache Rat Check Failures on DISCLAIMER.bin.txt
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-206'>DL-206</a>] -         Delete log doesn&#39;t delete log segments
+</li>
+</ul>
+                            
+####Improvement
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-204'>DL-204</a>] -         Bump libthrift to latest version for distributedlog-core
+</li>
+</ul>
+                                                                            
+####Task
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-2'>DL-2</a>] -         DistributedLog should work with the official apache bookkeeper
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-81'>DL-81</a>] -         Build the distributedlog release procedure
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-82'>DL-82</a>] -         Automatically build and update the website
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-199'>DL-199</a>] -         Be able to support filesystem-path like name
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-205'>DL-205</a>] -         Remove StatusCode dependency on DLException
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/DL-207'>DL-207</a>] -         DL Graduation Item
+</li>
+</ul>
+                    
+
+### Github
+
+- [https://github.com/apache/distributedlog/milestone/1](https://github.com/apache/distributedlog/milestone/1)
```

Comment:
we might change this to show the closed issues - https://github.com/apache/distributedlog/milestone/1?closed=1

Reply:
Thank. will change it.

Reply:
Thanks. will change it.